client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -118,11 +106,6 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortE
                       </MuiAlert>
                     </Box>
                   )}
-                {isGraduatingBeforeCohortEndDate && (
-                  <MuiAlert severity='warning'>
```

Comment:
I think I said to keep this?

Reply:
Oh I misread that message, I'll add this back!